### ISSUE TITLE
I need the 'Colletion Type' to be brought back on one of the tabs in the Download Queue

### ISSUE THREAD

**Issue by 0rkaM** [justin025/onthespot#43]
### Feature Description

Right now as of OnTheSpot v1.0.0rc7 there is not anymore the column for the 'Collection Type'; I would need the 'Collection Type' column in the Downloads tab to be back.

### Justification

When that column was available, I was able to see from which playlist were the files from; that helped me see how many were completed, remaining, failed of each playlist... which is important to me to see how many files I have to retry, or find sometime somewhere else to complete a specific set. Since I have multiple playlist sets on which I work on any given time and I place from 5 to 10 playlist for download at a time, having that column would allow me to start working on a playlist set as soon as that playlist is done downloading without waiting for all of them to be downloaded.
Thank you

### Additional Information

This is a vital function for me, since I almost exclusively work on various playlist at a time and almost never on single files. Without this information the program loses most of its usefulness and I would have to go back to the original, or alternative forks.
 
Maybe would be nice to be able to select some of the column available for visibility, including 'Collection Type' so who needs it can activate it and who does not can set it off.

**Comment by justin025** [comment]
If you want to go back to the original no one is stopping you lol, 0.7.1also has collection types if this is a must have feature. The reason I have not got to this yet is because I have spent some time thinking about the columns and how I should implement customization.

Nonetheless I have readded this tab in the latest commit and will have a test build out your way once the other two issues are ironed out.

![2024-10-28_06-55_1](https://github.com/user-attachments/assets/7fce7b82-1704-41af-85bb-b3c0f0c97d28)

**Comment by 0rkaM** [comment]
I didn't mean that as an offence, I just try to fill in the 'Justification' and the 'Additional Information' to explain why that is needed and what my option are. I am actually using your last build and also v.0.71, together with a script using Zotify depending on what I am working on, but I would prefer use only one tool: yours... and now that I think about it could be cool if at a later date you could add a CLI and/or API interface... but maybe is better to leave that to Zotify or a fork of it instead of making this too much of a Swiss Army knife approach... I am split on this suggestion.
Anyway, thank you for adding that column back and as I said if it bothers someone it could be made optional, so peps that do not need or want can select to make it not visible.
Cheers.

**Comment by justin025** [comment]
I don't get offended easily don't worry, just a reminder that 1.0.0 although stable (and in my opinion much more feature complete than 0.7.1) it is still just a release candidate and undergoing internal changes.

There already is a basic cli I wrote for another user who requested it, the instructions are in the discord but I can forward them here if you'd like. 

Column was added to RC-8, let me know if it works for you and if so feel free to close the issue.
